code/game/objects/items/toys.dm

```diff
@@ -1169,6 +1170,26 @@
 		user << "<span class='alert'>Nothing happens.</span>"
 
 /*
+ * Snowballs
+ */
+
+/obj/item/toy/snowball
+	name = "snowball"
+	desc = "A compact ball of snow. Good for throwing at people."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "snowball"
+	throwforce = 12 //pelt your enemies to death with lumps of snow

```

Comment:
Its not meant to be a real 'toy', unless being inside the toys.dm is enough to qualify it for an arcade toy prize. I couldn't think of any other decent spot for it and they're hard to get anyway.